### PR TITLE
Adding mutatest Python package

### PR DESCRIPTION
**Pull request by EvanKepner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example): **yes, included in tarball, no txt extension.**
- [x] Source is from official source: **yes, PyPI**
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged): **No vendoring.**
- [x] If static libraries are linked in, the license of the static library is packaged.: **No static libraries used.**
- [x] Build number is 0: **Yes**
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details): **Yes, using PyPI `mutatest` 2.1.2 tarball link.**
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there:**Yes, I am the only maintainer.**

### CODE REVIEW

**Comment by EvanKepner** [comment]
@conda-forge-admin, please ping team

I'm a new contributor requesting PR review to add the Python package `mutatest` v2.1.3.

File: recipes/mutatest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - coverage >=4.4,<6.0
+    - pytest
+    - freezegun
+    - hypothesis
+    - pytest-cov
+  source_files:
+    - pytest.ini
+  commands:
+    - pytest --pyargs mutatest
+
```
Comment (npavlovikj) [inline recipes/mutatest/meta.yaml]: Can you also add a test for the command-line execution of `mutatest`, e.g. `mutatest -h`, https://conda-forge.readthedocs.io/en/latest/testing.html#command-line-utilities?

Comment (EvanKepner) [reply recipes/mutatest/meta.yaml]: @npavlovikj thanks for the quick review and the pointer, done in the latest commit and passed.

**Comment by npavlovikj** [comment]
@EvanKepner , this is a pretty good first recipe - thank you for contributing!

**Pull request merged**